tests/x11/gnome_music.pm

```diff
@@ -10,13 +10,23 @@
 # - Close gnome-music
 # Maintainer: Max Lin <mlin@suse.com>
 
-use base "x11test";
+use base "opensusebasetest";
 use strict;
 use warnings;
 use testapi;
 use utils;
 
+sub check_bsc1206793 {
+    # Due to bsc#1206793, the test may fail with package version at '41.1-150400.3.3.1'
+    select_console 'root-console';
+    zypper_call('in gnome-music');
```

Comment:
IMHO, this is duplicate with the `if script_output('rpm -q gnome-music') =~ '41.1-150400.3.3.1'`.
The aim of `sub check_bsc1206793` is to get the gnome-music version and then decide if we should record a soft failure, am I right?

Reply:
Yes, we have seen the issue with gnome-music pkg at this version, so add a workaround here to "softfail" this test. and I will revert this commit once the bug fix is checked in.